File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/RolePermissionId.java
package org.dddml.wms.domain;


public class RolePermissionId
{
    private String roleId;

    public String getRoleId()
    {
        return this.roleId;
    }

    public void setRoleId(String roleId)
    {
        this.roleId = roleId;
    }

    private String permissionId;

    public String getPermissionId()
    {
        return this.permissionId;
    }

    public void setPermissionId(String permissionId)
    {
        this.permissionId = permissionId;
    }

    RolePermissionId ()
    {
    }

    public RolePermissionId (String roleId, String permissionId)
    {
        this.roleId = roleId;
        this.permissionId = permissionId;
    }

}

File: Dddml.Wms.AdminUI/ts/RolePermissionId.ts
namespace Dddml.Wms.Domain {

	export class RolePermissionId {

		private roleId: string;

		getRoleId(): string {
			return this.roleId;
		}

		setRoleId(roleId: string) {
			this.roleId = roleId;
		}

		private permissionId: string;

		getPermissionId(): string {
			return this.permissionId;
		}

		setPermissionId(permissionId: string) {
			this.permissionId = permissionId;
		}

	}
}

